components/Banner/examples/tests/snapshot/MDCBannerSnapshotTests.m

```diff
@@ -161,6 +166,7 @@ - (void)testLongTextWithTwoActionsRTL {
   [button2 setTitleColor:UIColor.blackColor forState:UIControlStateNormal];
   button2.uppercaseTitle = YES;
   [self changeViewToRTL:self.bannerView];
+  self.bannerView.imageView.hidden = YES;
 
   // Then
   [self generateSnapshotAndVerifyForView:self.bannerView];
```

Comment:
Please move these tests to the tests/ directory and configure them according to our component testing conventions (BUILD targets namely).

Reply:
Those were moved to the expected directory. I made these changes first in the original one to make it work and then copy these to the right directory (https://github.com/material-components/material-components-ios/pull/7065/files#diff-62e09c98ab764ee3afe12efbb897870b) for Github review because of the reason you mentioned before about Github diff.

Reply:
I will delete the original files after review and before merge.